nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/main/java/org/apache/nifi/web/security/ProxiedEntitiesUtils.java

```diff
@@ -148,13 +147,13 @@ public static String buildProxiedEntitiesChainString(final NiFiUser user) {
         return StringUtils.join(proxyChain, "");
     }
 
-    public static void successfulAuthorization(HttpServletRequest request, HttpServletResponse response, Authentication authResult) {
+    public static void successfulAuthentication(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
Are these correct?:
  

```
/**
* If using a proxy, tell the proxy the proxied user identity was authenticated.
*
* @param request The original client request that failed to be authenticated.
* @param response Servlet response to the client containing the successful authentication details.
*/
```

    /**
     * If using a proxy, give the proxy an explanation why authentication failed.
     *
     * @param request The proxied client request that failed to be authenticated.
     * @param response Servlet response to the proxied client containing the unsuccessful authentication attempt details.
     * @param failed The related exception thrown and explanation for the unsuccessful authentication attempt.
     */

Reply:
I don't understand the `@param request The original client request that failed to be authenticated` for `successfulAuthentication` -- do you mean that the incoming request wasn't authenticated because it came from the proxy? I am looking for method comments that explain _why_ and _when_ this method gets called.

Reply:
Made a mistake with that first comment:

/**
* If a successfully authenticated request was made via a proxy, relevant proxy headers will be added to the response.
*
* @param request The proxied client request that was successfully authenticated.
* @param response A servlet response to the client containing the successful authentication details.
*/
public static void successfulAuthentication(HttpServletRequest request, HttpServletResponse response) {